From 2c854f62d645724a93d4cf711b7a0b6dae7fb9af Mon Sep 17 00:00:00 2001
From: MarkusVolk <f_l_k@t-online.de>
Date: Thu, 25 Nov 2021 06:41:03 +0100
Subject: [PATCH] Makefile: replace sdl-config with pkg-config

Signed-off-by: MarkusVolk <f_l_k@t-online.de>
---
 quakespasm/Quake/Makefile | 16 ++++++----------
 1 file changed, 6 insertions(+), 10 deletions(-)

diff --git a/Makefile b/Makefile
index 9304ed61..da264dac 100644
--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ DEBUG   ?= 0
 
 CC ?= gcc
 LINKER = $(CC)
-
+PKG_CONFIG ?= pkg-config
 STRIP ?= strip
 
 #CPUFLAGS= -mtune=i686
@@ -102,18 +102,14 @@ endif
 
 ifeq ($(USE_SDL2),1)
 CFLAGS += -DUSE_SDL2
+SDL_CFLAGS ?= $(shell $(PKG_CONFIG) --silence-errors --cflags sdl2)
+SDL_LIBS ?= $(shell $(PKG_CONFIG) --silence-errors --libs sdl2)
+CFLAGS += $(SDL_CFLAGS)
+LDFLAGS = $(SDL_LIBS)
 endif
 #make use of LFS on 32bit even on systems.
 CFLAGS += -D_FILE_OFFSET_BITS=64
 
-ifeq ($(USE_SDL2),1)
-SDL_CONFIG ?= sdl2-config
-else
-SDL_CONFIG ?= sdl-config
-endif
-SDL_CFLAGS := $(shell $(SDL_CONFIG) --cflags)
-SDL_LIBS   := $(shell $(SDL_CONFIG) --libs)
-
 ifeq ($(HOST_OS),sunos)
 NET_LIBS   :=-lsocket -lnsl -lresolv
 else
-- 
2.25.1

