From 62383b4308d1e4250d6b3d227dd2c0eae4c2b1e0 Mon Sep 17 00:00:00 2001
From: MarkusVolk <f_l_k@t-online.de>
Date: Sun, 24 Oct 2021 16:29:36 +0200
Subject: [PATCH] remove SOURCE_ASM

Signed-off-by: MarkusVolk <f_l_k@t-online.de>
---
 Makefile.common | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile.common b/Makefile.common
index dbe25b6f..f1abc0b8 100644
--- a/Makefile.common
+++ b/Makefile.common
@@ -475,9 +475,6 @@ ifeq ($(HAVE_NEON),1)
 	SOURCES_CXX   += $(VIDEODIR_GLIDEN64)/src/Neon/3DMathNeon.cpp \
 						  $(VIDEODIR_GLIDEN64)/src/Neon/gSPNeon.cpp
 
-	SOURCES_ASM += $(LIBRETRO_COMM_DIR)/audio/conversion/float_to_s16_neon.S \
-						$(LIBRETRO_COMM_DIR)/audio/conversion/s16_to_float_neon.S \
-						$(LIBRETRO_COMM_DIR)/audio/resampler/drivers/sinc_resampler_neon.S
 else
 	SOURCES_CXX   += $(VIDEODIR_GLIDEN64)/src/3DMath.cpp
 endif
-- 
2.25.1

