From 1b448c79d9fe3836af220868c0854ede905b19c0 Mon Sep 17 00:00:00 2001
From: MarkusVolk <f_l_k@t-online.de>
Date: Wed, 20 Oct 2021 01:39:11 +0200
Subject: [PATCH] fix

Signed-off-by: MarkusVolk <f_l_k@t-online.de>
---
 Makefile | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 3d2b37c3..0053aede 100644
--- a/Makefile
+++ b/Makefile
@@ -518,7 +518,7 @@ endif
 include Makefile.common
 
 ifeq ($(HAVE_NEON), 1)
-   COREFLAGS += -DHAVE_NEON -D__ARM_NEON__ -D__NEON_OPT -ftree-vectorize -mvectorize-with-neon-quad -ftree-vectorizer-verbose=2 -funsafe-math-optimizations -fno-finite-math-only
+   COREFLAGS += -DHAVE_NEON -D__ARM_NEON__ -D__NEON_OPT -ftree-vectorize -ftree-vectorizer-verbose=2 -funsafe-math-optimizations -fno-finite-math-only
 endif
 
 ifeq ($(LLE), 1)
-- 
2.25.1

