From 859904fd1fcee121cd36d3195a56f3cc6cea38c4 Mon Sep 17 00:00:00 2001
From: MarkusVolk <f_l_k@t-online.de>
Date: Fri, 12 Nov 2021 04:16:11 +0100
Subject: [PATCH] meson.build: add missing files

Signed-off-by: MarkusVolk <f_l_k@t-online.de>
---
 meson.build | 211 ++++++++++++++++++++++++++++++++++++++++++++++++++--
 1 file changed, 205 insertions(+), 6 deletions(-)

diff --git a/meson.build b/meson.build
index 711d639a22..7c2a79d92e 100644
--- a/meson.build
+++ b/meson.build
@@ -1365,7 +1365,16 @@ client_common = static_library('fc_client_common',
   )
 
 install_data('data/helpdata.txt',
+             'data/cimpletoon.tilespec',
+             'data/isophex.tilespec',
+             'data/isotrident.tilespec',
+             'data/toonhex.tilespec',
+             'data/alio.tilespec',
+             'data/trident.tilespec',
              'data/hexemplio.tilespec',
+             'data/hex2t.tilespec',
+             'data/amplio.tilespec',
+             'data/amplio2.tilespec',
              'data/stdmusic.musicspec',
              'data/stdsounds.soundspec',
              install_dir : join_paths(get_option('datadir'), 'freeciv'))
@@ -1384,9 +1393,13 @@ install_data('data/misc/colors.tilespec',
              'data/misc/specialists.spec',
              'data/misc/events.png',
              'data/misc/events.spec',
+             'data/misc/buildings.png',
+             'data/misc/buildings.spec',
              'data/misc/buildings-large.spec',
              'data/misc/wonders-large.spec',
+             'data/misc/flags.spec',
              'data/misc/flags-large.spec',
+             'data/misc/shields.spec',
              'data/misc/shields-large.spec',
              'data/misc/cursors.png',
              'data/misc/cursors.spec',
@@ -1447,6 +1460,21 @@ install_data('data/buildings/airport.png',
              install_dir : join_paths(get_option('datadir'),
                                       'freeciv/buildings'))
 
+install_data('data/scenarios/british-isles.sav',
+             'data/scenarios/earth-large.sav',
+             'data/scenarios/earth-small.sav',
+             'data/scenarios/europe.sav',
+             'data/scenarios/europe_1900_WWI.sav',
+             'data/scenarios/france.sav',
+             'data/scenarios/hagworld.sav',
+             'data/scenarios/iberian-peninsula.sav',
+             'data/scenarios/italy.sav',
+             'data/scenarios/japan.sav',
+             'data/scenarios/north_america.sav',
+             'data/scenarios/tileset-demo.sav',
+             'data/scenarios/tutorial.sav',
+             install_dir : join_paths(get_option('datadir'), 'freeciv/scenarios'))
+
 install_data('data/wonders/apollo_program.png',
              'data/wonders/asmiths_trading_co.png',
              'data/wonders/colossus.png',
@@ -2060,7 +2088,9 @@ foreach flag : flags
 endforeach
 
 install_data('data/override/flags-large.spec',
+             'data/override/flags.spec',
              'data/override/shields-large.spec',
+             'data/override/shields.spec',
              install_dir : join_paths(get_option('datadir'), 'freeciv/override'))
 
 install_data('data/hexemplio/activities.png',
@@ -2105,16 +2135,177 @@ install_data('data/hexemplio/activities.png',
              'data/hexemplio/water3.spec',
              install_dir : join_paths(get_option('datadir'), 'freeciv/hexemplio'))
 
-install_data('data/amplio2/units.png',
-             'data/amplio2/units.spec',
-             'data/amplio2/extra_units.png',
-             'data/amplio2/extra_units.spec',
+install_data('data/hex2t/grid.png',
+             'data/hex2t/grid.spec',
+             'data/hex2t/items.png',
+             'data/hex2t/items.spec',
+             'data/hex2t/overlays.png',
+             'data/hex2t/overlays.spec',
+             'data/hex2t/select.png',
+             'data/hex2t/select.spec',
+             'data/hex2t/tiles.png',
+             'data/hex2t/tiles.spec',
+             'data/hex2t/unitcost.png',
+             'data/hex2t/unitcost.spec',
+             install_dir : join_paths(get_option('datadir'), 'freeciv/hex2t'))
+
+install_data('data/isophex/darkness.png',
+             'data/isophex/darkness.spec',
+             'data/isophex/grid.png',
+             'data/isophex/grid.spec',
+             'data/isophex/rivers.png',
+             'data/isophex/rivers.spec',
+             'data/isophex/terrain1.png',
+             'data/isophex/terrain1.spec',
+             'data/isophex/terrain2.png',
+             'data/isophex/terrain2.spec',
+             install_dir : join_paths(get_option('datadir'), 'freeciv/isophex'))
+
+install_data('data/trident/cities.png',
+             'data/trident/cities.spec',
+             'data/trident/earth.png',
+             'data/trident/earth.spec',
+             'data/trident/explosions.png',
+             'data/trident/explosions.spec',
+             'data/trident/extra_units.png',
+             'data/trident/extra_units.spec',
+             'data/trident/fog.png',
+             'data/trident/fog.spec',
+             'data/trident/auto_ll.spec',
+             'data/trident/grid.png',
+             'data/trident/grid.spec',
+             'data/trident/roads.png',
+             'data/trident/roads.spec',
+             'data/trident/select.png',
+             'data/trident/select.spec',
+             'data/trident/tiles.png',
+             'data/trident/tiles.spec',
+             'data/trident/units.png',
+             'data/trident/units.spec',
+             install_dir : join_paths(get_option('datadir'), 'freeciv/trident'))
+
+install_data('data/isotrident/cities.png',
+             'data/isotrident/cities.spec',
+             'data/isotrident/fog.png',
+             'data/isotrident/fog.spec',
+             'data/isotrident/grid.png',
+             'data/isotrident/grid.spec',
+             'data/isotrident/morecities.png',
+             'data/isotrident/morecities.spec',
+             'data/isotrident/nuke.png',
+             'data/isotrident/nuke.spec',
+             'data/isotrident/select.png',
+             'data/isotrident/select.spec',
+             'data/isotrident/terrain1.png',
+             'data/isotrident/terrain1.spec',
+             'data/isotrident/terrain2.png',
+             'data/isotrident/terrain2.spec',
+             'data/isotrident/tiles.png',
+             'data/isotrident/tiles.spec',
+             'data/isotrident/unitcost.png',
+             'data/isotrident/unitcost.spec',
+             'data/isotrident/unitextras.png',
+             'data/isotrident/unitextras.spec',
+             install_dir : join_paths(get_option('datadir'), 'freeciv/isotrident'))
+
+install_data('data/cimpletoon/orient_units.spec',
+             'data/cimpletoon/orient_units.png',
+             install_dir : join_paths(get_option('datadir'), 'freeciv/cimpletoon'))
+
+install_data('data/alio/burrowtubes.png',
+             'data/alio/burrowtubes.spec',
+             'data/alio/fortresses.png',
+             'data/alio/fortresses.spec',
+             'data/alio/hills.png',
+             'data/alio/hills.spec',
+             'data/alio/riversbrown.png',
+             'data/alio/riversbrown.spec',
+             'data/alio/riversgreen.png',
+             'data/alio/riversgreen.spec',
+             'data/alio/roads.spec',
+             'data/alio/roads.png',
+             'data/alio/terrain.png',
+             'data/alio/terrain.spec',
+             'data/alio/tunnels.png',
+             'data/alio/tunnels.spec',
+             install_dir : join_paths(get_option('datadir'), 'freeciv/alio'))
+
+install_data('data/amplio/ancientcities.png',
+             'data/amplio/ancientcities.spec',
+             'data/amplio/explosions.png',
+             'data/amplio/explosions.spec',
+             'data/amplio/fog.png',
+             'data/amplio/fog.spec',
+             'data/amplio/grid.png',
+             'data/amplio/grid.spec',
+             'data/amplio/maglev.png',
+             'data/amplio/maglev.spec',
+             'data/amplio/medievalcities.png',
+             'data/amplio/medievalcities.spec',
+             'data/amplio/moderncities.png',
+             'data/amplio/moderncities.spec',
+             'data/amplio/nuke.png',
+             'data/amplio/nuke.spec',
+             'data/amplio/ocean.png',
+             'data/amplio/ocean.spec',
+             'data/amplio/select-alpha.png',
+             'data/amplio/select.spec',
+             'data/amplio/terrain1.png',
+             'data/amplio/terrain1.spec',
+             'data/amplio/terrain2.png',
+             'data/amplio/terrain2.spec',
+             'data/amplio/tiles.png',
+             'data/amplio/tiles.spec',
+             'data/amplio/units.png',
+             'data/amplio/units.spec',
+             'data/amplio/upkeep.png',
+             'data/amplio/upkeep.spec',
+             'data/amplio/water.png',
+             'data/amplio/water.spec',
+             install_dir : join_paths(get_option('datadir'), 'freeciv/amplio'))
+
+install_data('data/amplio2/activities.png',
+             'data/amplio2/activities.spec',
              'data/amplio2/animals.png',
              'data/amplio2/animals.spec',
-             'data/amplio2/nuke.png',
-             'data/amplio2/nuke.spec',
+             'data/amplio2/bases.png',
+             'data/amplio2/bases.spec',
+             'data/amplio2/cities.png',
+             'data/amplio2/cities.spec',
              'data/amplio2/explosions.png',
              'data/amplio2/explosions.spec',
+             'data/amplio2/extra_units.png',
+             'data/amplio2/extra_units.spec',
+             'data/amplio2/fog.png',
+             'data/amplio2/fog.spec',
+             'data/amplio2/grid.png',
+             'data/amplio2/grid.spec',
+             'data/amplio2/hills.png',
+             'data/amplio2/hills.spec',
+             'data/amplio2/maglev.png',
+             'data/amplio2/maglev.spec',
+             'data/amplio2/mountains.png',
+             'data/amplio2/mountains.spec',
+             'data/amplio2/nuke.png',
+             'data/amplio2/nuke.spec',
+             'data/amplio2/ocean.png',
+             'data/amplio2/ocean.spec',
+             'data/amplio2/select-alpha.png',
+             'data/amplio2/select.spec',
+             'data/amplio2/terrain1.png',
+             'data/amplio2/terrain1.spec',
+             'data/amplio2/terrain2.png',
+             'data/amplio2/terrain2.spec',
+             'data/amplio2/tiles.png',
+             'data/amplio2/tiles.spec',
+             'data/amplio2/units.png',
+             'data/amplio2/units.spec',
+             'data/amplio2/upkeep.png',
+             'data/amplio2/upkeep.spec',
+             'data/amplio2/veterancy.png',
+             'data/amplio2/veterancy.spec',
+             'data/amplio2/water.png',
+             'data/amplio2/water.spec',
              install_dir : join_paths(get_option('datadir'), 'freeciv/amplio2'))
 
 install_data('data/stdmusic/CullamBruce-Lockhart--Dawning_Fanfare.ogg',
@@ -2142,6 +2333,14 @@ install_data('data/stdsounds/foot3.ogg',
 
 endif
 
+i = ['16x16', '32x32', '48x48', '64x64', '128x128']
+foreach iconsize : i
+    install_data(join_paths('data/icons/', iconsize, 'freeciv-client.png'),
+                 join_paths('data/icons/', iconsize, 'freeciv-server.png'),
+                 join_paths('data/icons/', iconsize, 'freeciv-modpack.png'),
+                 install_dir : join_paths(get_option('datadir'), 'icons/hicolor', iconsize, 'apps'))
+endforeach
+
 gtk322_dep = dependency('gtk+-3.0', version : '>= 3.22', required : false)
 gtk4_dep = dependency('gtk4', version : '>= 4.0.0', required : false)
 
-- 
2.25.1

