From 9c54eebb95ca7b703edd98a20539bac252694454 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Thu, 1 Apr 2021 14:38:21 +0200
Subject: [PATCH] generic-pc: increase standard resolution

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 libgeneric-pc/init.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgeneric-pc/init.cpp b/libgeneric-pc/init.cpp
index 1447eb3..bdd2106 100644
--- a/libgeneric-pc/init.cpp
+++ b/libgeneric-pc/init.cpp
@@ -19,7 +19,7 @@ void hal_api_init()
 		hal_debug_init();
 	hal_info("%s begin, initialized=%d, debug=0x%02x\n", __func__, (int)initialized, debuglevel);
 	if (! glfb) {
-		int x = 1280, y = 720; /* default OSD FB resolution */
+		int x = 1920, y = 1080; /* default OSD FB resolution */
 		/*
 		 * export GLFB_RESOLUTION=720,576
 		 * to restore old default behviour
-- 
2.25.1

