From 8b9672265fef2f04370e5db8fc70ff964713e6b9 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sat, 22 May 2021 18:57:56 +0200
Subject: [PATCH] CMakeLIST.txt: dont install /run entry

Signed-off-by: Markus Volk <f_l_k@t-online.de>

---
 CMakeLists.txt | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4cc3c4dce88..30ea8e0cb98 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -557,12 +557,12 @@ if(ENABLE_APPDATA_FILE AND ENABLE_GAME)
 	install(FILES packaging/org.wesnoth.Wesnoth.appdata.xml DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/metainfo )
 endif()
 
-if(ENABLE_SERVER AND FIFO_DIR)
-	install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory \$ENV{DESTDIR}/${FIFO_DIR})")
-	if(SERVER_UID AND SERVER_GID)
-		install(CODE "execute_process(COMMAND chown ${SERVER_UID}:${SERVER_GID} \$ENV{DESTDIR}/${FIFO_DIR})")
-	endif()
-endif()
+#if(ENABLE_SERVER AND FIFO_DIR)
+#	install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory \$ENV{DESTDIR}/${FIFO_DIR})")
+#	if(SERVER_UID AND SERVER_GID)
+#		install(CODE "execute_process(COMMAND chown ${SERVER_UID}:${SERVER_GID} \$ENV{DESTDIR}/${FIFO_DIR})")
+#	endif()
+#endif()
 
 #
 # uninstall
