From 8c0e2bb06d2c4c1c254d28509a3b699b76617a9a Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Fri, 12 Mar 2021 15:56:56 +0100
Subject: [PATCH] lakka.h: adjust path

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 lakka.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lakka.h b/lakka.h
index f098a1080c..b04bb02e4d 100644
--- a/lakka.h
+++ b/lakka.h
@@ -19,10 +19,10 @@
 #define __RARCH_LAKKA_H
 
 #define LAKKA_SSH_PATH       "/storage/.cache/services/sshd.conf"
-#define LAKKA_SAMBA_PATH     "/storage/.cache/services/samba.conf"
+#define LAKKA_SAMBA_PATH     "/etc/samba/smb.conf"
 #define LAKKA_BLUETOOTH_PATH "/storage/.cache/services/bluez.conf"
 #define LAKKA_UPDATE_DIR     "/storage/.update/"
-#define LAKKA_CONNMAN_DIR    "/storage/.cache/connman/"
+#define LAKKA_CONNMAN_DIR    "/home/retro/.config/connman"
 #define LAKKA_LOCALAP_PATH   "/storage/.cache/services/localap.conf"
 
 #include "switch_performance_profiles.h"
-- 
2.25.1

