From 6be5264906e5568dd857a2cbc3a21907284aff5a Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sat, 3 Apr 2021 10:05:56 +0200
Subject: [PATCH] neutrino.cpp: adjust resolution for generic hardware

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 src/neutrino.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/neutrino.cpp b/src/neutrino.cpp
index d30eaffca..cb65bf2e9 100644
--- a/src/neutrino.cpp
+++ b/src/neutrino.cpp
@@ -400,7 +400,7 @@ int CNeutrinoApp::loadSetup(const char * fname)
 
 	// video
 	int vid_Mode_default = VIDEO_STD_720P50;
-#if HAVE_ARM_HARDWARE
+#if HAVE_ARM_HARDWARE || HAVE_GENERIC_HARDWARE
 	vid_Mode_default = VIDEO_STD_1080P50;
 #elif HAVE_CST_HARDWARE && defined(BOXMODEL_CST_HD2)
 	vid_Mode_default = VIDEO_STD_1080P24;
@@ -482,7 +482,7 @@ int CNeutrinoApp::loadSetup(const char * fname)
 		g_settings.enabled_video_modes[7] = 1; // VIDEO_STD_1080I50
 #if HAVE_CST_HARDWARE && defined(BOXMODEL_CST_HD2)
 		g_settings.enabled_video_modes[10] = 1; // VIDEO_STD_1080P24
-#elif HAVE_ARM_HARDWARE
+#elif HAVE_ARM_HARDWARE || HAVE_GENERIC_HARDWARE
 		g_settings.enabled_video_modes[13] = 1; // VIDEO_STD_1080P50
 		g_settings.enabled_video_modes[18] = 1; // VIDEO_STD_2160P50
 #endif
@@ -813,7 +813,7 @@ int CNeutrinoApp::loadSetup(const char * fname)
 	g_settings.recording_stream_subtitle_pids  = configfile.getBool("recordingmenu.stream_subtitle_pids", true);
 	g_settings.recording_stream_pmt_pid        = configfile.getBool("recordingmenu.stream_pmt_pid"      , false);
 	g_settings.recording_filename_template     = configfile.getString("recordingmenu.filename_template" , "%C_%T_%d_%t");
-#if HAVE_ARM_HARDWARE || HAVE_MIPS_HARDWARE
+#if HAVE_ARM_HARDWARE || HAVE_MIPS_HARDWARE
 	g_settings.recording_bufsize               = configfile.getInt32("recording_bufsize", 4);
 	g_settings.recording_bufsize_dmx           = configfile.getInt32("recording_bufsize_dmx", 2);
 #endif
@@ -930,7 +930,7 @@ int CNeutrinoApp::loadSetup(const char * fname)
 
 	//screen configuration
 	int osd_res = OSDMODE_720;
-#if HAVE_ARM_HARDWARE || (HAVE_CST_HARDWARE && defined(BOXMODEL_CST_HD2))
+#if HAVE_ARM_HARDWARE || (HAVE_CST_HARDWARE && defined(BOXMODEL_CST_HD2)) || HAVE_GENERIC_HARDWARE
 	osd_res = OSDMODE_1080;
 #endif
 	g_settings.osd_resolution      = (osd_resolution_tmp == -1) ? configfile.getInt32("osd_resolution", osd_res) : osd_resolution_tmp;
-- 
2.25.1

